src/run/parser/livesplit.rs

```diff
@@ -209,7 +209,7 @@ fn parse_segment<R: BufRead>(
                                 *segment.comparison_mut(&comparison) = t;
                             })?;
                         }
-                        run.add_custom_comparison(comparison);
+                        run.add_custom_comparison(comparison).ok();
```

Comment:
? operator, probably needs a new error and .map_err(|_| Error::InvalidComparisonName).